docs/getting_started/DatamodelWorkflow.md

```diff
@@ -0,0 +1,158 @@
+# Getting Started
+
+Start by creating your own extensible, distributed platform that:
+
+* Implements a datamodel as K8s CRDs.
+
+* Generates accessors that faciliate easy consumption of the datamodel.
+
+* Bootstrap a cloud native application that consumes that datamodel and is ready-to-go in no time.
+
+
+## Datamodel Workflow
+
+Nexus Datamodel provides the framework, language, toolkit and runtime to implement state/data required by application, that is:
+
+* hierarchical
+* distributed
+* consistent
+* customizable
+
+Nexus datamodel framework supports formalization of objects/nodes and their relationship in a spec that is expressed using Golang syntax and is fully compliant with the Golang compiler.
+
+**NOTE: If you would prefer to follow a pre-cooked Datamodel, as you try this workflow, execute commands from the code sections below** 
+
+This guided workflow will walk you through setting up a datamodel that is local to your application or Importable datamodel
+
+* [Nexus Install](DatamodelWorkflow.md#nexus-install)
+
+* [Nexus Pre-req verify](DatamodelWorkflow.md#nexus-pre-req-verify)
+
+* [Setup Datamodel Workspace](DatamodelWorkflow.md#setup-datamodel-workspace)
+
+* [Datamodel Build](DatamodelWorkflow.md#datamodel-build)
+  
+* [Datamodel Install](DatamodelWorkflow.md#datamodel-install)
+  
+* [Datamodel Playground](DatamodelWorkflow.md#datamodel-playground)
+
+
+## Nexus Install
+
+Install Nexus CLI
+
+```
+curl -LJ https://github.com/vmware-tanzu/graph-framework-for-microservices/releases/latest/download/nexus-$(uname -s | awk '{print tolower($0)}')_$(go env GOARCH)  -o nexus && chmod +x nexus
```

Comment:
this downloads cli to local dir, so `nexus` command can't be used outside of this dir, right? we should move to user path, for example /usr/local/bin

Reply:
or `"$(go env GOPATH)"/bin`

Reply:
Initially, we thought about this.  Should we add this @xmen4xp

Reply:
Will track this change with this ticket : https://jira.eng.vmware.com/browse/NPT-641

Reply:
for now, we retain `go install` way. removed curl-based download.